packages/core/saved-objects/core-saved-objects-migration-server-internal/src/actions/bulk_overwrite_transformed_documents.ts

```diff
@@ -26,6 +26,12 @@ export interface BulkOverwriteTransformedDocumentsParams {
   index: string;
   operations: BulkOperation[];
   refresh?: estypes.Refresh;
+  /**
+   * If true, we prevent Elasticsearch from auto-creating the index if it
+   * doesn't exist. We use the ES paramater require_alias: true so `index`
+   * must be an alias, otherwise the bulk index will fail.
+   */
+  useAliasToPreventAutoCreate?: boolean;
```

Comment:
NIT: personal opinion, but given it's a 1-1 mapping to an ES option and it's only used internally, I would have just called this `requireAlias`.

Reply:
yeah I was going back and forth... this `requireAlias` "hack" / workaround / trick is so weird that it felt like I need to explain it because we don't actually care about using an alias...

Reply:
Yeah, I would have gone with the actual option name + your tsdoc to explain why it should be used, but I'm fine with both